### ISSUE TITLE
Output for Rubeus monitor task is not shown in the grunt console

### ISSUE THREAD

**Issue by 3xpl01tc0d3r** [cobbr/Covenant#223]
**Bug**

While running `Rubeus monitor /interval:5` task I don't see any output of the command. 

**To Reproduce**
1. Go to any existing Grunt and click Interact Tab
2. Run the Rubeus task - Command - Rubeus monitor /interval:5 or Rubeus monitor /interval:1

**Expected behavior**
The output of the Rubeus task should be shown in the grunt console.

**Screenshots**

![image](https://user-images.githubusercontent.com/33144026/90339264-47adf180-e00d-11ea-8ab0-e6c288d93e4b.png)

**Covenant Server Information:**
 - OS: Windows 10

**Browser Information:**
 - Browser Chrome
 - Version 84

**Target Information (System that implant is running on):**
 - OS: Server 2016
 - Version 1607

**Comment by cobbr** [comment]
@3xpl01tc0d3r My suspicion here would be that it's not outputting any information, because Rubeus doesn't have any new tickets to report from the Monitor module.

Are you confident that there is any information that it should be returning? Is Rubeus returning information when used outside of Covenant?

**Comment by 3xpl01tc0d3r** [comment]
Yes I am sure that it should return the output. It works fine when I run Rubeus outside Covenant and I get proper output.
Just to give you more info, I am abusing the Unconstrained Delegation by using SpoolSample.exe. But when I run Rubeus monitor task in Covenant I can't see the output.

**Comment by cobbr** [comment]
Hey @3xpl01tc0d3r , i'm not 100% certain, but I believe some recent changes to streaming output may have addressed this. Do you mind checking to confirm if this fixes your issue?

**Comment by hjtenklooster** [comment]
Hey @cobbr , I ran into the same issue as @3xpl01tc0d3r , but it does not seem to be resolved by the commits 490096fb71cf3e8de14fe4d45c4e51db8c31c231 & 7555b19ffb9401c0e37094c25e404a640b1688d7.

I added an extra line in the beginning with Console.WriteLine("Test1");. However that also doesn't end up when using the monitor command, nor does the Rubeus logo show.

The commands that run only for a few seconds however work, and return output as soon as they are finished.
